tests/debian_test.yaml

```diff
@@ -1,4 +1,14 @@
 schemaVersion: '2.0.0' # Make sure to test the latest schema version
+
+driver: "docker"
```

Comment:
I've added this, but i'm not sure how we should handle this in code. 

It feels to me like this should be the default, and overridable by the command line flag. Let me know your thoughts on how to proceed with this.

Reply:
Currently, the driver gets passed in as a CLI flag and then instantiated here: https://github.com/GoogleCloudPlatform/container-structure-test/blob/master/structure_test.go#L111

Instead, that will have to happen in the `Parse()` method, similar to how the `schemaVersion` gets parsed out of the config file: https://github.com/GoogleCloudPlatform/container-structure-test/blob/master/structure_test.go#L80

You should still be able to reuse the logic for `InitDriverImpl` here.